app/components/header.tsx

```diff
@@ -1,11 +1,73 @@
-import { Text, Center } from '@chakra-ui/react';
+import {
+  Text,
+  Heading,
+  Avatar,
+  Flex,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  IconButton,
+} from '@chakra-ui/react';
+import { TriangleUpIcon, LockIcon } from '@chakra-ui/icons';
+import { Link, useFetcher } from '@remix-run/react';
+
+import { useUser } from '~/utils';
 
 export default function Header() {
+  const user = useUser();
+  const fetcher = useFetcher();
   return (
-    <header>
-      <Center>
-        <Text fontSize="2xl">Hi! I am the header</Text>
-      </Center>
-    </header>
+    <Flex
+      as="header"
+      backgroundColor="brand.500"
+      height="auto"
+      alignItems="center"
+      shadow="2xl"
+      fontSize="18"
+      paddingY="2"
+    >
+      <HStack width="50%" color="white" justifyContent="flex-start" gap="10" marginLeft="10">
+        <Link to={{ pathname: '/domains' }}>
+          <Flex alignItems="center">
+            <TriangleUpIcon marginRight="2" />
+            <Text>Domains</Text>
+          </Flex>
+        </Link>
+
+        <Link to={{ pathname: '/certificate' }}>
+          <Flex alignItems="center">
+            <LockIcon marginRight="2" />
+            <Text>Certificate</Text>
+          </Flex>
+        </Link>
+      </HStack>
+      <Link to={{ pathname: '/' }}>
```

Comment:
I think we should remove this, since the home page will be the domain page anyway. Feels kinda redundant with the domain link. What do you think?

Reply:
Wouldn't we have to replace it later on, because the domain page will became our new index page? Or do we just redirect them to the domain page when they're at `/`?

Reply:
Yeah, we will probably need to change the index page to redirect to the domain page. I just mean that the logo linking to anything is kinda redundant. It could just be plain text

Reply:
oh that's what you meant, yeah sure I'll remove it